### PR TITLE
LPD-36730 Home page has locales appended to its sitemap's URLs when the property locale.prepend.friendly.url.style is set to 0

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPD-36730

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:report:13643901

**Comment by MrJohn1911** [comment]
ci:report:13664381

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplAlternateURLTest.java [hunk 1]
```diff
@@ -114,6 +115,55 @@ public static void tearDownClass() {
...
+
+		_testAlternateURLWithHomeLayout(
+			availableLocales, defaultLocale, themeDisplay, 0);
+
+		_testAlternateURLWithHomeLayout(
+			availableLocales, defaultLocale, themeDisplay, 1);
+
+		_testAlternateURLWithHomeLayout(
+			availableLocales, defaultLocale, themeDisplay, 2);
+
```
Comment (jonathanmccann) [inline modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/PortalImplAlternateURLTest.java:162]: The new lines between the calls to `_testAlternateURLWithHomeLayout` should be removed to match the formatting of the rest of the class.

**Review by jonathanmccann — COMMENTED** [review]
Everything looks good other than the one formatting issue. After that, if the tests are passing, feel free to forward.

**Comment by MrJohn1911** [comment]
No test failures related to the pull, last push was just to remove lines between the  `_testAlternateURLWithHomeLayout` method.

CC: @jonathanmccann

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:foward

**Comment by MrJohn1911** [comment]
ci:forward

**Comment by MrJohn1911** [comment]
ci:forward

**Comment by MrJohn1911** [comment]
ci:report:15926634

**Comment by MrJohn1911** [comment]
ci:help

**Comment by MrJohn1911** [comment]
ci:forward:force

**Pull request closed without merging**